plugins/modules/system/ufw.py

```diff
@@ -54,6 +54,8 @@
     description:
       - Insert the corresponding rule as rule number NUM.
       - Note that ufw numbers rules starting with 1.
+      - If I(delete=true) and a value is provided for I(insert),
+        then the task is biased for the rule to be deleted.
```

Comment:
I would make it more explicit:
```suggestion
        then I(insert) is ignored.
```